doc/bakugo_spec.md

```diff
@@ -529,13 +529,11 @@ In addition to explicit blocks in the source code, there are implicit blocks:
 Blocks nest and influence _scoping_.
 
 # Declarations and scope
-| NOTE |
-| :--- |
-| Changes yet to made |
 
 A _declaration_ binds a non-_blank_ identifier to a _constant_, _type_, _variable_, _function_. Every
-identifier in a program must be declared. No identifier may be declared twice in the same block, and no
-identifier may be declared in both the file and package block.
+identifier in a program must be declared. No identifier may be declared in both the file and package block.
```

Comment:
```suggestion
identifier in a program must be declared.
```
Since we don't support _packages_ in this spec. I think we can remove it safely.